bin/pytorch_inference/Main.cc

```diff
@@ -198,6 +199,10 @@ int main(int argc, char** argv) {
 
     // Reduce memory priority before installing system call filters.
     ml::core::CProcessPriority::reduceMemoryPriority();
+    if (lowPriority == true) {
+        LOG_DEBUG(<< "Running with low priority");
+        ml::core::CProcessPriority::reduceCpuPriority();
```

Comment:
In the other programs we do this after connecting the named pipes, because named pipe timeouts stop the process working altogether. For example see https://github.com/elastic/ml-cpp/blob/59242db5f3e7666ddcd08397f9367e5e9bf1f4b4/bin/autodetect/Main.cc#L173-L176